keras/applications/efficientnet.py

```diff
@@ -354,23 +389,31 @@ def round_repeats(repeats):
 
     # Build stem
     x = img_input
-    x = layers.Rescaling(1.0 / 255.0)(x)
-    x = layers.Normalization(axis=bn_axis)(x)
-    if weights == "imagenet":
-        # Note that the normaliztion layer uses square value of STDDEV as the
+
+    if lite:
+        x = layers.Normalization(mean=127.0, variance=128.0**2)(x)
```

Comment:
We should use the same preprocessing as the below, for consistency. This means that the Normalization layer should be adapted to account for the scaling, and the normalization state should be part of the weights file.

Reply:
@fchollet 
Given these models will be in a separate class / file I was thinking about adding `include_preprocessing` option, as it is in newer implementations, like EfficientNetV2 or ResNet-RS.

Putting the Normalization parameters will be incompatible with this option as the `Normalization` layer will always have to be present. 

Perhaps instead of putting the normalization parameters in the weights file, the `include_preprocessing` option could be used?

Please let me know!

Reply:
> Perhaps instead of putting the normalization parameters in the weights file, the include_preprocessing option could be used?

Sounds good to me!

Reply:
Added `include_preprocessing` option.